webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlTransformer.scala

```diff
@@ -244,6 +265,7 @@ object SparqlTransformer {
 
       case _ =>
         // Is the statement in KnoraExplicitNamedGraph?
+        // TODO-BL: see if explicit named graph is still needed, now that we don't support GraphDB anymore.
```

Comment:
Can you resolve or care about this TODO now?

Reply:
I removed the comment, as I have noted it elsewhere